src/components/workshops/CarouselProject.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component, Fragment } from 'react'
+import api from 'api'
+import styled from 'styled-components'
+import { isURL } from 'validator'
+import {
+  Box,
+  Flex,
+  Label,
+  Heading,
+  Link as A,
+  Text,
+  Image,
+  Button,
+  theme,
+} from '@hackclub/design-system'
+
+const ProjectOuter = styled(Flex).attrs({
+  p: [0, 0, 4],
+  bg: [theme.colors.white, theme.colors.white, theme.colors.white],
+  justify: 'flex-end',
+  flexDirection: 'column',
+})`
+  position: relative;
+  flex-basis: 0;
+  flex-grow: 1;
+  flex-shrink: 1;
+  overflow: hidden;
+  ${theme.mediaQueries.md} {
+    align-items: stretch;
+  }
+`
+
+const ProjectOuterLeft = styled(ProjectOuter)`
+  border-radius: 10px 0px 0px 10px;
+  ${theme.mediaQueries.md} {
+    border-radius: 20px 5px 5px 20px;
+  }
+`
+
+const ProjectOuterRight = styled(ProjectOuter)`
+  border-radius: 0px 10px 10px 0px;
+  ${theme.mediaQueries.md} {
+    border-radius: 5px 20px 20px 5px;
+  }
+`
+
+const TextBar = styled(Flex).attrs({
+  px: [1, 1, 0],
+  py: 0,
+  justify: ['center', 'center', 'space-between'],
```

Comment:
Can use `null` if you don't want to change the previous breakpoint:
```suggestion
  justify: ['center', null, 'space-between'],
```